include/struct_decls/struct_0202B628_decl.h

```diff
@@ -1,6 +1,6 @@
 #ifndef POKEPLATINUM_STRUCT_0202B628_DECL_H
 #define POKEPLATINUM_STRUCT_0202B628_DECL_H
 
-typedef struct UnkStruct_0202B628_t UnkStruct_0202B628;
+typedef struct UnkStruct_0202B628_t Journal;
```

Comment:
Can we merge this struct into another include somewhere? (and the others as well)